### PR TITLE
Adds a new anti-Brute medication called "Meralyne."

### PR DESCRIPTION
**Pull request by Lainwright-Lazuli**
Provides a second tier for healing Brute damage, similar to Dexalin+ and oxygen damage. Credit to Hughgent for the idea.

The recipe for meralyne if you want to test it out is 1 inaprovaline, 1 bicaridine, and 1 iron per part / u. Overdoses at 15u, and has double the overdose damage of bicard.

[why]: # We have dermaline as an upgrade to kelotane (burn damage), dexalin + for dexalin (oxygen damage), clonexadone for cryoxadone (genetic damage). 

:cl: Lainwright Lazuli
add: A new drug called Meralyne, more effective at treating brute damage at cost of a more punishing overdose
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry_reagents/medical.dm [hunk 1]
```diff
@@ -627,6 +627,26 @@ datum/reagent/medicine/synaptizine/overdose_crit_process(mob/living/M, alien)
...
+	id = "meralyne"
+	description = "Meralyne is a concentrated form of bicardine and can be used to treat extensive blunt trauma."
+	color = "#E6666C"
+	overdose_threshold = REAGENTS_OVERDOSE
+	overdose_crit_threshold = REAGENTS_OVERDOSE_CRITICAL
+	scannable = TRUE
+
+/datum/reagent/medicine/meralyne/on_mob_life(mob/living/M, alien)
+	M.heal_limb_damage(4*REM,0)
+	..()
```
Comment (DominikPanic) [inline code/modules/reagents/chemistry_reagents/medical.dm]: Swap these two lines and change it to
```suggestion
	. = ..()
```

File: code/modules/reagents/chemistry_reagents/medical.dm [hunk 2]
```diff
@@ -627,6 +627,26 @@ datum/reagent/medicine/synaptizine/overdose_crit_process(mob/living/M, alien)
...
+	name = "Meralyne"
+	id = "meralyne"
+	description = "Meralyne is a concentrated form of bicardine and can be used to treat extensive blunt trauma."
+	color = "#E6666C"
+	overdose_threshold = REAGENTS_OVERDOSE
+	overdose_crit_threshold = REAGENTS_OVERDOSE_CRITICAL
+	scannable = TRUE
+
+/datum/reagent/medicine/meralyne/on_mob_life(mob/living/M, alien)
+	M.heal_limb_damage(4*REM,0)
```
Comment (DominikPanic) [inline code/modules/reagents/chemistry_reagents/medical.dm]: Optional, but good whitespace is nice.
```suggestion
	M.heal_limb_damage(4 * REM, 0)
```

File: code/modules/reagents/chemistry_reagents/medical.dm (hunk 1 above)
Comment (Lainwright-Lazuli) [reply code/modules/reagents/chemistry_reagents/medical.dm]: Sadly, this causes an error where the compiler works, but the client gets massive runtime errors with this change. The old version worked upon reversion.

File: code/modules/reagents/chemistry_reagents/medical.dm (hunk 2 above)
Comment (Lainwright-Lazuli) [reply code/modules/reagents/chemistry_reagents/medical.dm]: Committed before I saw the "commit suggestion" tab. Thanks.

File: code/modules/reagents/chemistry_reagents/medical.dm (hunk 1 above)
Comment (Rohesie) [reply code/modules/reagents/chemistry_reagents/medical.dm]: What kind of runtime errors?

Comment (Lainwright-Lazuli) [reply code/modules/reagents/chemistry_reagents/medical.dm]: I can send you part of the log I had earlier. It's in #coding in the Discord, but part of it was that it somehow caused `runtime error: Cannot read null.squads` from changing the line as above.

Comment (Lainwright-Lazuli) [reply code/modules/reagents/chemistry_reagents/medical.dm]: Closed the connection even after a successful, "error-free" compilation.

Comment (Rohesie) [reply code/modules/reagents/chemistry_reagents/medical.dm]: That makes no sense, regarding this change. Try it again.

**Comment by Rohesie** [comment]
This has a ton of merge garbage. You'd be better off saving the changes somewhere, hard resetting to current master, and then re-applying the changes.

File: code/modules/reagents/chemistry_reagents/medical.dm (hunk 1 above)
Comment (Lainwright-Lazuli) [reply code/modules/reagents/chemistry_reagents/medical.dm]: Alright, after clearing out the merge garbage with a fresh install, the changes seem to have worked as smooth as butter as of a preliminary test.

**Pull request merged**